### PR TITLE
Code 169

### PR DESCRIPTION
**Pull request by AkagiShigeru**

### CODE REVIEW

File: sous_chef/sous_chef/messaging/todoist_api.py [hunk 1]
```diff
@@ -302,7 +303,7 @@ def _get_active_labels(self):
         return label_dict
 
     @staticmethod
-    def _get_due_date_str(due_date: datetime.datetime) -> str:
+    def _get_due_date_str(due_date: datetime.datetime) -> Union[str, None]:
```
Comment (ArBridgeman) [inline sous_chef/sous_chef/messaging/todoist_api.py:306]: as discussed, there should be a check or something that ensures only stings go here.

File: sous_chef/sous_chef/messaging/todoist_api.py [hunk 2]
```diff
@@ -30,6 +30,19 @@ def __str__(self):
         return f"{self.message}: tag={self.tag} for value={self.value}"
 
 
+@dataclass
```
Comment (ArBridgeman) [inline sous_chef/sous_chef/messaging/todoist_api.py:33]: As mentioned by developer, this should be taken out for a future commit.

**Review by ArBridgeman — CHANGES_REQUESTED** [review]

File: sous_chef/sous_chef/messaging/todoist_api.py (hunk 2 above)
Comment (AkagiShigeru) [reply sous_chef/sous_chef/messaging/todoist_api.py]: Removed for now

File: sous_chef/sous_chef/messaging/todoist_api.py (hunk 1 above)
Comment (AkagiShigeru) [reply sous_chef/sous_chef/messaging/todoist_api.py]: Changed that behaviour and corresponding tests, pls have a look.

**Pull request merged**